plaso/analysis/browser_search.py

```diff
@@ -308,26 +338,25 @@ def CompileReport(self, mediator):
     analysis_report.report_dict = results
     return analysis_report
 
-  def ExamineEvent(self, mediator, event):
+  def ExamineEvent(self, mediator, event, event_data):
     """Analyzes an event.
 
     Args:
       mediator (AnalysisMediator): mediates interactions between
           analysis plugins and other components, such as storage and dfvfs.
-      event (EventObject): event to examine.
+      event (EventObject): event.
+      event_data (EventData): event data.
     """
-    # This event requires an URL attribute.
-    url = getattr(event, 'url', None)
-    if not url:
+    if event_data.data_type not in self._SUPPORTED_EVENT_DATA_TYPES:
       return
 
-    # TODO: refactor this the source should be used in formatting only.
-    # Check if we are dealing with a web history event.
-    source, _ = formatters_manager.FormattersManager.GetSourceStrings(event)
-
-    if source != 'WEBHIST':
+    url = getattr(event_data, 'url', None)
+    if not url:
       return
 
+    parser_or_plugin_name = getattr(event_data, 'parser', 'N/A')
+    parser_or_plugin_name = getattr(event_data, 'plugin', parser_or_plugin_name)
```

Comment:
This can be removed, plugin isn't set any more, it's in the parser attribute.

Reply:
ack, removing